packages/commerce-sdk-react/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## v1.2.0-preview.0 (Nov 30, 2023)
+- Add StorefrontPreview component 'onContextChange' property.  [#1527](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1527)
```

Comment:
docs site is down for maintenance now. 😞 is this change documented in the storefront preview guide?